### DISCUSSION TITLE
Feature suggestion: single-binary deployment

### DISCUSSION THREAD

**Discussion by iameli** [grafana/grafana#45806] [General]
I'm working on a "batteries included" distribution of Livepeer + MistServer for folks that want a single command to run the entire stack. This is pretty easy with most go programs — we've got all our core microservices set up as single binaries, and we even bundle Traefik and Victoria Metrics! But Grafana is a bit more challenging. I can work around most of the configuration file dependencies with environment variables, but the `public` directory is impossible to dodge, and will have to be distributed alongside all of the binaries to get a working distribution.

It'd be awesome if it were possible to build a version of Grafana that could run as a single `grafana-server` binary with no external dependencies.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
